recipes/picrust2/meta.yaml

```diff
@@ -19,21 +19,21 @@ requirements:
     - pip
     - python >=3.5,<3.7
   run:
-    - biom-format 2.1.7
+    - biom-format >=2.1.7
     - epa-ng 0.3.5
-    - gappa 0.2.0
+    - gappa 0.4.0
     - glpk 4.65
-    - h5py 2.9.0
+    - h5py >=2.9.0
     - hmmer 3.2.1
-    - joblib 0.13.1
-    - numpy 1.16.3
-    - pandas 0.24.2
+    - joblib >=0.13.1
+    - numpy >=1.16.2
+    - pandas >=0.24.2
     - python >=3.5,<3.7
```

Comment:
Does this really not work in python 3.7?

Reply:
Thanks for the review! It does appear to work, but this warning comes up a lot when you run the whole pipeline with Python 3.7:

`
/home/gavin/local/miniconda3/envs/dev_picrust2/lib/python3.7/site-packages/biom/table.py:185: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
    from collections import defaultdict, Hashable, Iterable
`

I'd prefer not to support Python 3.7 until I figure out how to avoid that warning. I'll troubleshoot that further when putting together the next release.

Reply:
OK.